pubsub/pubsub.go

```diff
@@ -40,3 +54,26 @@ func Ping(pubsub PubSub) error {
 		return fmt.Errorf("ping is not implemented by this pubsub")
 	}
 }
+
+// DefaultBulkMessager is default implemnetation for BukMessager
+type DefaultBulkMessager struct {
+	p PubSub
+}
+
+// NewDefaultBulkMessager to create new DefaultBulkMessager for a PubSub
+func NewDefaultBulkMessager(pubsub PubSub) DefaultBulkMessager {
+	return DefaultBulkMessager{
+		p: pubsub,
+	}
+}
+
+// BulkPublish Default Implementation
+func (p *DefaultBulkMessager) BulkPublish(req *BulkPublishRequest) (BulkPublishResponse, error) {
+	return BulkPublishResponse{}, nil
```

Comment:
should this be unimplemented for now in this PR?

Reply:
Yes, I will create a follow-up PR after this.

Reply:
can change error conveying unimplemented